operating_unit/models/res_users.py

```diff
@@ -53,14 +53,34 @@ def _compute_operating_unit_ids(self):
                     dom = [
                         "|",
                         ("company_id", "=", False),
-                        ("company_id", "in", self.env.context["allowed_company_ids"]),
+                        (
+                            "company_id",
+                            "in",
+                            self.env.context["allowed_company_ids"],
+                        ),
```

Comment:
revert this, unneeded change

Reply:
@MiquelRForgeFlow  I don't get this change exactly. Can you elaborate more?